.github/workflows/router-ci.yaml

```diff
@@ -145,9 +145,6 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-        credentials:
```

Comment:
Hi @git-hulk, the docker credentials are used to avoid rate limit issues.

Reply:
I see, does it sound good to change the DockerHub to other mirror sources instead of failing the integration, which is quite unfriendly for external contributors? please refer to https://github.com/actions/runner-images/issues/1445.

Reply:
This is a good question and we also thought about it but it's not a priority for us right now. Pointing to a docker mirror can be a security risk. We need to evaluate different solutions.